plugins/gui/src/GUIWindows.cpp

```diff
@@ -722,221 +739,225 @@ bool megamol::gui::GUIWindows::ConsumeTriggeredScreenshot(void) {
 
 bool megamol::gui::GUIWindows::SynchronizeGraphs(megamol::core::MegaMolGraph* megamol_graph) {
 
-    // 1) Load all known calls from core instance ONCE ---------------------------
-    if (!this->configurator.GetGraphCollection().LoadCallStock(core_instance)) {
-        megamol::core::utility::log::Log::DefaultLog.WriteError(
-            "[GUI] Failed to load call stock once. [%s, %s, line %d]\n", __FILE__, __FUNCTION__, __LINE__);
-        return false;
-    }
-    /// TODO Load all known modules from core instance ONCE
-    /// XXX Omitted since this task takes ~2 seconds and would always block megamol for this period at start up!
+    if (this->state.gui_pre_drawing_enabled) {
```

Comment:
Suggesting early exit here and with the other `if (drawing_enabled)` conditionals, too.
```
if (! drawing_enabled) {
    return;
}

// draw GUI
```

Reply:
Thanks, see commit 4720be1